runner/src/client/sass/application.scss

```diff
@@ -85,3 +85,19 @@
     }
   }
 }
+
+.govuk-list {
```

Comment:
GOV.UK Frontend follows BEM (loosely). 

it would be
 
```
.govuk-list {
  &--multi-page { // -- denotes "modifier"
     //..
  }
}
```

and use the class like 

`<ul class="govuk-list govuk-list--multi-page">`

.govuk-list applies the "base" styling, and .govuk-list--multi-page applies anything new.